cloud/linode/cloud.go

```diff
@@ -25,14 +26,46 @@ const (
 // We expect it to be initialized with flags external to this package, likely in
 // main.go
 var Options struct {
-	KubeconfigFlag *pflag.Flag
-	LinodeGoDebug  bool
+	KubeconfigFlag        *pflag.Flag
+	LinodeGoDebug         bool
+	EnableRouteController bool
+	VPCName               string
 }
 
+// vpcDetails is set when VPCName options flag is set.
+// We use it to list instances running within the VPC if set
+type vpcDetails struct {
+	mu   sync.RWMutex
```

Comment:
This was just to keep code safe just in case some gotest when run in parallel invokes init in multiple goroutines. I agree it doesn't and will most likely never be, was one of the suggestions in earlier reviews just to make global variable thread safe and hence used mutex here as well.